ngi_visualizations/fpkm_scatter/fpkm_scatter.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/python
+"""
+fpkm_scatter.py
+
+Takes FPKM counts from two conditions and makes a scatter plot.
+Also calculates a r-squared correlation score.
+
+Either takes two Cufflinks FPKM summary files or a summary file
+with multiple samples.
+"""
+
+from __future__ import print_function
```

Comment:
This is some python 3 forward-compatability thing? But I don't see that you use print as a statement in your code, only once as a function.

Very cool though, I didn't know about this.